src/RecoveryServices/RecoveryServices.SiteRecovery/NetworkInterfaceCard/NewAzureRmAsrVmNicConfig.cs

```diff
@@ -294,23 +203,19 @@ public override void ExecuteSiteRecoveryCmdlet()
                         return;
                     }
 
-                    if (!this.MyInvocation.BoundParameters.ContainsKey(
-                            Utilities.GetMemberName(() => this.RecoveryVMNetworkId)))
-                    {
-                        this.RecoveryVMNetworkId =
-                            this.ReplicationProtectedItem.SelectedRecoveryAzureNetworkId;
-                    }
-
-                    if (!this.MyInvocation.BoundParameters.ContainsKey(
-                            Utilities.GetMemberName(() => this.RecoveryVMSubnetName)))
+                    if (this.IPConfigs != null)
                     {
-                        this.RecoveryVMSubnetName = vmNic.RecoveryVMSubnetName;
+                        if (!validateAndPopulateIPConfigs(vmNic))
```

Comment:
method name - Pascal case instead of camel

Reply:
updated